controllers/idler/owners.go

```diff
@@ -24,8 +24,13 @@ func newOwnerFetcher(discoveryClient discovery.ServerResourcesInterface, dynamic
 	}
 }
 
+type objectWithGVR struct {
+	object client.Object
+	gvr    *schema.GroupVersionResource
```

Comment:
This doesn't seem to be used.

Reply:
good catch, this is gonna be used in the upcoming PR :-) stay tuned :wink: